corehq/project_limits/rate_limiter.py

```diff
@@ -158,12 +161,24 @@ def __init__(self, per_user_rate_definition, constant_rate_definition=None):
         self.constant_rate_definition = constant_rate_definition or RateDefinition()
 
     def get_rate_limits(self, domain):
-        n_users = get_n_users_for_rate_limiting(domain)
-        return (
-            self.per_user_rate_definition
-            .times(n_users)
-            .plus(self.constant_rate_definition)
-        ).get_rate_limits()
+        domain_users = get_n_users_in_domain(domain)
+        enterprise_users = get_n_users_in_subscription(domain)
+        limit_pairs = [
+            (domain_users, domain),
+            (enterprise_users, _get_account_name(domain))
+        ]
+        old_enterprise_calculation = get_n_users_old_enterprise_count(domain)
+        if old_enterprise_calculation is not None:
+            limit_pairs.append((old_enterprise_calculation, f'old_enterprise:{domain}'))
```

Comment:
why do we still need the old enterprise bucket?

Reply:
We wanted to roll this out in a way that would not result in changes to the formula on the off chance that some enterprises were  within the old limit but in excess of the new ones. This gives us a chance to see how the changes will affect users before they start actually limiting.